pkg/api/middleware/authorization.go

```diff
@@ -119,22 +123,29 @@ func (a *TagAuthorizer) authorizeFindingOp(findingID, tag string) error {
 		return errNotFound
 	}
 
-	if !contains(f.Target.Tags, tag) {
+	if !contains(f.Target.Teams, teamID) {
 		return errForbidden
 	}
 
 	return nil
 }
 
-func (a *TagAuthorizer) authorizeTagOp(path, authTag string) error {
-	reqTag := strings.TrimPrefix(path, tagOpPrefix)
-	if reqTag != authTag {
+func (a *TeamAuthorizer) authorizeTeamOp(path, authTeam string) error {
+	reqTeam := strings.TrimPrefix(path, teamOpPrefix)
```

Comment:
That endpoint that you used as an example does not exist now. However, the idea is to create it in the next commits, so I will address it and update the filter accordingly so that it also matches this case.